src/layout/DefaultLayout/DefaultLayout.tsx

```diff
@@ -29,7 +34,14 @@ const DefaultLayout = ({
     { path: '/ussf-documentation', label: 'USSF documentation' },
   ]
 
-  const { data } = useGetUserQuery()
+  const { data }: PortalUser | any = useGetUserQuery()
+
+  useEffect(() => {
+    setMongoUserInfo(data)
+    if (data) {
+      setTheme(data.theme)
+    }
+  }, [data])
```

Comment:
Great question. Yeah I think we can add this to maybe the article and page layouts. Not sure if it'd be useful to add to the error and login layouts. I just opted for default layout because I think that covers most of the app and I'm not sure if a user can even land on a page other than MySpace after authenticating, correct?